data/tlg2016/tlg001/__cts__.xml

```diff
@@ -11,11 +11,11 @@
 			Cambridge University Press. 1891.</ti:description>
 	</ti:edition>
 	
-	<ti:edition urn="urn:cts:greekLit:tlg2016.tlg001.1st1K-lat1" 
+	<ti:translation urn="urn:cts:greekLit:tlg2016.tlg001.1st1K-lat1" 
```

Comment:
Is this a translation? I thought there were both original Latin and Greek versions of the Passion (not sure the Latin is a translation of the Greek or vice versa).

Reply:
I only know because I did a project on it, but this is probably more akin to Bodin, where we have two editions/versions. There was a lost Latin original (we presume).